src/schema/sale/index.js

```diff
@@ -319,6 +320,19 @@ export const SaleType = new GraphQLObjectType({
         },
       },
       registration_ends_at: date,
+      registrationStatus: {
+        type: Bidder.type,
+        description: "A registration for this sale",
+        resolve: (
+          { id },
+          _args,
+          _request,
+          { rootValue: { meBiddersLoader } }
+        ) => {
+          if (!meBiddersLoader) return null
+          return meBiddersLoader({ sale_id: id }).then(([bidder]) => bidder)
```

Comment:
I’m assuming that there can only be one item in this list when scoping to a sale?

Reply:
Thanks 👍